app/controllers/api/v1/tickets_controller.rb

```
@@ -1,311 +1,424 @@
+            end
+            
+            # Then verify they belong to the specified team
+            unless team.users.exists?(id: assignee.id)
+              raise ActiveRecord::RecordNotFound, "User #{params[:assignee_id]} not found in team #{team.id}"
+            end
+            
+            @ticket.assignee = assignee
+            @ticket.status = 'assigned'
+          end
+        end
+      end
```

RuboCop (1.73)

[convention] Assignment Branch Condition size for process_team_and_assignee is too high. [<5, 22, 5> 23.11/23]

(Metrics/AbcSize)